.travis.yml

```diff
@@ -5,10 +5,13 @@ services:
 - docker
 env:
   global:
-  - PINS="colombe:."
+  - PINS="colombe:. sendmail:. sendmail-lwt:."
   - DISTRO="debian-stable"
   - TESTS=true
   matrix:
-  - PACKAGE="colombe" OCAML_VERSION="4.04.2"
-  - PACKAGE="colombe" OCAML_VERSION="4.05.0"
-  - PACKAGE="colombe" OCAML_VERSION="4.06.0"
+  - PACKAGE="colombe" OCAML_VERSION="4.07.0"
```

Comment:
will be quicker if you replace `4.07.0` with `4.07` (no patch number) so it can use the prebuilt containers. We don't include the patch number since we assume you want to test the latest patch release